addons/fleet/models/fleet_vehicle.py

```diff
@@ -123,6 +127,12 @@ def _compute_service_activity(self):
             activities_state = set(state for state in vehicle.log_services.mapped('activity_state') if state and state != 'planned')
             vehicle.service_activity = sorted(activities_state)[0] if activities_state else 'none'
 
+    @api.depends('model_id')
+    def _compute_image_1920(self):
+        for vehicle in self:
+            if not vehicle.image_1920:
+                vehicle.image_1920 = vehicle.model_id.image_1920
```

Comment:
Why not a related ? Is it really important to keep track of the vehicle image at the time ?